### PR TITLE
Fingerprint: KIA Niro HEV 2021 - South Korea model

### PR DESCRIPTION
**Pull request by kangtae1**
KIA Niro HEV 2021 - South Korea model (MY2019)

bd6b3c208fa1b9ef|2024-02-17--18-08-19

### CODE REVIEW

**Comment by sshane** [comment]
| Dongle, platform, VIN                                                                                                                | Name from VIN 🆚 CarInfo                                                                    | Segments                  | Bad seg tags   | Good seg tags                                            |
|:-------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------|:--------------------------|:---------------|:---------------------------------------------------------|
| [bd6b3c208fa1b9ef](https://useradmin.comma.ai/?onebox=bd6b3c208fa1b9ef)<br><sub>KIA NIRO HYBRID 2021<br>KNACE81CG........</sub>      | KIA  1990 🆚<br>Kia Niro Hybrid 2022<br><sub>🎉 <b>New model year!</b> 🎉</sub>             | 78 routes, 3856 segments  |                | - all lat active: 1775<br>- no input all lat active: 844 |

File: selfdrive/car/hyundai/fingerprints.py [hunk 1]
```diff
@@ -1224,20 +1224,24 @@
   CAR.KIA_NIRO_HEV_2021: {
     (Ecu.engine, 0x7e0, None): [
       b'\xf1\x816H6G5051\x00\x00\x00\x00\x00\x00\x00\x00',
+      b'\xf1\x816H6F6051\x00\x00\x00\x00\x00\x00\x00\x00',
     ],
     (Ecu.transmission, 0x7e1, None): [
       b'\xf1\x816U3J9051\x00\x00\xf1\x006U3H1_C2\x00\x006U3J9051\x00\x00HDE0G16NL3\x00\x00\x00\x00',
       b'\xf1\x816U3J9051\x00\x00\xf1\x006U3H1_C2\x00\x006U3J9051\x00\x00HDE0G16NL3\xb9\xd3\xfaW',
+      b'\xf1\x816U3J7051\x00\x00\xf1\x006U3H1_C2\x00\x006U3J7051\x00\x00HDE0G16KL1}\x1f\xad\xa4',
```
Comment (sshane) [inline selfdrive/car/hyundai/fingerprints.py:1232]:
```suggestion
      b'\xf1\x006U3H1_C2\x00\x006U3J7051\x00\x00HDE0G16KL1}\x1f\xad\xa4',
```

use shorter

**Comment by sshane** [comment]
| Dongle, platform, VIN                                                                                                           | Name from VIN 🆚 CarDocs             | Segments                                    | Bad seg tags   | Good seg tags                                                                           |
|:--------------------------------------------------------------------------------------------------------------------------------|:-------------------------------------|:--------------------------------------------|:---------------|:----------------------------------------------------------------------------------------|
| [bd6b3c208fa1b9ef](https://useradmin.comma.ai/?onebox=bd6b3c208fa1b9ef)<br><sub>KIA NIRO HYBRID 2021<br>KNACE81CG........</sub> | KIA  1990 🆚<br>Kia Niro Hybrid 2022 | 85 routes,<br>3825 segments<br>(63.8 hours) |                | - valid torque params: 3127<br>- all lat active: 1927<br>- no input all lat active: 748 |

**Comment by sshane** [comment]
Which harness did you use? @kangtae1?

**Comment by kangtae1** [comment]
@sshane Hyundai D

**Pull request merged**